app.js

```diff
@@ -0,0 +1,114 @@
+const hambuger = document.querySelector('#hambug');
+const closetog = document.querySelector('#togclose');
+const toggleMenu = document.querySelector('.toggle-menu');
+
+function show() {
+  toggleMenu.style.display = 'flex';
+  toggleMenu.style.top = '0';
+}
+
+function close() {
+  toggleMenu.style.top = '-100%';
+}
+
+hambuger.addEventListener('click', show);
+closetog.addEventListener('click', close);
+
+document.querySelectorAll('.list-menu').forEach((link) => link.addEventListener('click', close));
+
+// Dynamic Speaker Section
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.

Reply:
I removed the commented lines